### DISCUSSION TITLE
Support binding on top of macOS Sequoia window snapping shortcuts

### DISCUSSION THREAD

**Discussion by injust** [ghostty-org/ghostty#3118] [Feature Requests, Ideas] [answered]
macOS Sequoia introduced new window snapping shortcuts:

- `ctrl+page_up/page_down/home/end` snaps the current window to the top/bottom/left/right half
- Adding `shift` to the above will _additionally_ snap the _previous_ window to the other half

However, it seems like applications can override these shortcuts. In Chrome for example:

- `ctrl+page_up/page_down` switches to the previous/next tab
- `ctrl+shift+page_up/page_down` moves the current tab to the left/right

I can set these Chrome-inspired bindings in Ghostty like so, but they seem to only trigger the window snapping:

```
keybind = ctrl+page_up=previous_tab
keybind = ctrl+page_down=next_tab
keybind = ctrl+shift+page_up=move_tab:-1
keybind = ctrl+shift+page_down=move_tab:1
```

Does Ghostty need to do something so it can override the system-wide window snapping bindings?

**Comment by mitchellh** [comment, marked as answer]
Dupe #3187
